### DISCUSSION TITLE
Incorrect dimension

### DISCUSSION THREAD

**Discussion by Thalvur** [domlysz/BlenderGIS#712] [Q&A]
First of all sorry for my bad english.

I have a problem. You may be able to reproduce it using the following method. Go google earth, make a line with the measuring tool 8km. Now make an 8km line in the same place in the Blender. Notice anything? The distance between two selected points is 297m in GoogleMaps, in Blender EXPORT_GOOGLE_SAT_WM it is 371m.  All GEO data is too big, although nothing is scaled anywhere. Use Blender 2.93.6 and 3.2.

Is the error with me or in the code?

And maybe you can add xyz and gml file import?

**Comment by domlysz** [comment]
https://github.com/domlysz/BlenderGIS/wiki/FAQ#why-my-measurements-on-basemaps-reference-seems-wrong-

**Reply by Thalvur** [reply]
Ah ok, thank you very much. that is explanatory. There doesn't seem to be any other way of importing, so after creating the projections I have to scale them all to get the desired size. I have a model from the local geodatabase, I didn't know if this was wrong or if the view of the map in Blender was wrong. Thanks a lot for the response. Incidentally, you do not need to register with OpenTopography. I signed up and got an API key. This was incorrect in the GIS even after several attempts. A look at the API script then shows this: demoapikeyot2022
This key also works without registering in the portal.
Thank you for this tool!

**Comment by jmeanxt** [comment]
experiencing the same.
